### PR TITLE
Add `hierarchicalforecast` to github workflows.

### PR DESCRIPTION
**Pull request by melopeo**
*Description of changes:*
add hierarchicalforecast to github workflow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by lostella** [comment]
I vote for merging this with the statsforecast one https://github.com/awslabs/gluonts/blob/dev/.github/workflows/tests-statsforecast.yaml

**Comment by melopeo** [comment]
Propose a name for it and we can do it. Nixtla, maybe? At the end these are wrappers from their repositories.

**Comment by lostella** [comment]
> Propose a name for it and we can do it. Nixtla, maybe? At the end these are wrappers from their repositories.

Yes, sounds good to me

File: .github/workflows/tests-nixtla.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: StatsForecast tests
+name: nixtla tests
```
Comment (lostella) [inline .github/workflows/tests-nixtla.yaml]:
```suggestion
name: Nixtla tests
```

just for consistency with other workflows 🙂

File: test/ext/hierarchicalforecast/require-packages.txt [hunk 2]
```diff
@@ -1 +1,2 @@
+statsforecast
```
Comment (lostella) [inline test/ext/hierarchicalforecast/require-packages.txt:1]: Is this needed @jaheba? How are these used?

**Comment by lostella** [comment]
Just the workflow name otherwise good to go! 🚀

**Review by lostella — APPROVED** [review]

File: test/ext/hierarchicalforecast/require-packages.txt (hunk 2 above)
Comment (jaheba) [reply test/ext/hierarchicalforecast/require-packages.txt:1]: I would assume this doesn't need it.

The idea behind `require-packages` is that tests in that folder are only run if the required packages are installed. Here, I would assume we only need `hierarchicalforecast` as a check.

**Pull request merged**